packages/firebase_remote_config/firebase_remote_config/ios/Classes/FLTFirebaseRemoteConfigPlugin.m

```diff
@@ -29,6 +45,15 @@ + (instancetype)sharedInstance {
 }
 
 - (instancetype)init {
+#if TARGET_OS_IPHONE
+  self.hostReachability = [Reachability reachabilityWithHostName:remoteHostName];
+  [self.hostReachability startNotifier];
+  [[NSNotificationCenter defaultCenter] addObserver:self
```

Comment:
Is there any drawback to attaching to the NSNotificationCenter? Like new permissions or some warnings that could be thrown during compilation?

Reply:
No warnings during compilation nor permissions required. You do get more logs regarding reachability.

Reply:
Great :)

Reply:
hmmmm, it doesn't throw a warning on the first build, but it does on subsequent builds 🤔. I guess I'll remove the observer. You're required to add a `NSBonjourServices` key to your `Info.plist`. That's a shame as that would've been nice for diagnosing the problem.